modules/base/files/puppet/97-last-puppet-run

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+# Script that reports when Puppet last ran, in bold if more than an hour ago.
+
+set -e
+
+PATH=/bin:/usr/bin:/usr/local/bin
+LANG=C
+CONFIG_VERSION=$(awk -F\" '/^configuration_version:/ {$1="";print}' /var/lib/puppet/state/last_run_report.yaml)
```

Comment:
Needs to be:

/opt/puppetlabs/server/data/puppetserver/state/last_run_report.yaml

instead of

/var/lib/puppet/state/last_run_report.yaml